irohad/consensus/yac/impl/yac_gate_impl.cpp

```diff
@@ -55,6 +55,9 @@ namespace iroha {
 
         if (not event.round_data) {
           current_block_ = boost::none;
+          // previous block is committed to block storage, it is safe to clear
+          // the cache
+          consensus_result_cache_->release();
```

Comment:
please add a TODO to use `onBlock()` subscription in `BlockLoaderService` instead of unneeded complications with `consensus_result_cache_`